.github/workflows/release-desktop-app.yml

```diff
@@ -124,12 +124,22 @@ jobs:
         run: yarn make-${{ matrix.spec.platform }}-${{ matrix.spec.arch }}
         working-directory: apps/electron
 
+      - name: make Linux (arm64)
+        if: ${{ matrix.spec.platform == 'macos' && matrix.spec.arch == 'arm64' }}
```

Comment:
You may also need to specify another entry at `matrix.spec` instead of adding a new step here
          - { os: ubuntu-latest, platform: linux, arch: arm64 }

Reply:
no github doesn’t support

Reply:
https://github.com/actions/runner-images/issues/5631

Reply:
Please check https://github.com/toeverything/AFFiNE/pull/2251/files#diff-a4a05576b431b4411a4fa6f76ea795f9b3aa8f4fb1b45fb7becce0a9d85a7a1fR92-R95

Reply:
> [actions/runner-images#5631](https://github.com/actions/runner-images/issues/5631)

Oh okay, so this means we can only build arm64 on Mac? I guess you can use this instead:

```yml
- { os: macos-latest, platform: linux, arch: arm64 }
```